core/templates/services/math-interactions.service.ts

```diff
@@ -84,7 +84,9 @@ export class MathInteractionsService {
     return errorMessage;
   }
 
-  validateExpression(expressionString: string, algebraic = true): boolean {
+  validateExpression(
+      expressionString: string, algebraic = true,
+      validVariablesList = []): boolean {
```

Comment:
Here and below: can we force clients to pass in this param rather than providing a default value? DItto for "algebraic".

In general, prefer explicit passing in of params, especially where there isn't an obvious default choice that would hold for the majority of cases.

Reply:
Got it. Done!